docs/user-documentation/glossary.md

```diff
@@ -1,95 +1,202 @@
  
 ## Fedora (Repository Software)
 Fedora is a digital asset management architecture upon which institutional repositories, digital archives, and digital library systems might be built. Fedora is the underlying architecture for a digital repository, and is not a complete management, indexing, discovery, and delivery application. 
  
 The Fedora repository functions as the standard _smart storage_ for Islandora.
  
 ## Field
-Data of a certain type that is attached to a content entity. For instance, on a Resource Node content type, you might have fields for a title, description, display hints, subjects, and other metadata. 
+Data of a certain type that is attached to a content entity. For instance, on a Resource Node [content type](#content-type), you might have fields for a title, description, display hints, subjects, and other metadata. 
  
+## Field instance
+
+## Field type
+
+## FITS
+File Information Tool Set, a set of software components for identifying, validating and extracting of technical metadata for a wide range of file formats.
+
+## Fixity
+Also file fixity; digital preservation term meaning that a digital file remains unchanged ('fixed') over time. Fixity checking verifies that a file has not been corrupted or manipulated during a transfer process or while being stored. Typically, a fixity checking process computes [checksums](#checksum) or cryptographic hashes for a file and compares the result to a reference value stored earlier. The contributed Islandora module 'RipRap' supports fixity checking and error reporting.
+
 ## Flysystem
-Flysystem is a filesystem abstraction library for PHP. Islandora uses Flysystem to swap about different backend filesystem applications. Islandora provides a custom Flysystem adapter for Fedora.
+Flysystem is a filesystem abstraction library for PHP. Islandora uses Flysystem to swap about different backend filesystem applications. Islandora provides a custom Flysystem adapter for [Fedora](#fedora-repository-software).
+
+## GLAM
+Acronym for "galleries, libraries, archives, and museums".
 
 ## IIIF
 The [International Image Interoperability Framework](https://iiif.io/). A set of open standards that help archives, libraries, and museums make the most of their digitized collections with deep zoom, annotation capabilities, and more, and also the community of users and developers that suport the framework.
  
 ## Ingest
-To ingest an object is to add an entry for it in Islandora. This is done primarily through the Drupal interface. 
+To ingest an object is to add an entry for it in Islandora. This is done primarily through the [Drupal](#drupal) interface. 
```

Comment:
Changed it to:

> This can be done through the [Drupal](#drupal) graphical user interface or one of the Drupal APIs ([REST](https://www.drupal.org/docs/drupal-apis/restful-web-services-api/restful-web-services-api-overview), [Migrate API](https://www.drupal.org/docs/drupal-apis/migrate-api/migrate-api-overview)). The third-party contributed software [Islandora Workbench](https://github.com/mjordan/islandora_workbench) uses the Drupal REST API for convenient bulk ingest.